README.md

```diff
@@ -6,10 +6,7 @@ This plugin provides client-side integration for the [CodePush service](http://c
 * [Supported React Native Platforms](#supported-react-native-platforms)
 * [Getting Started](#getting-started)
 * [iOS Setup](#ios-setup)
-    * [Plugin Installation](#plugin-installation-ios)
```

Comment:
Maybe we can just have a note about older releases and link to the readme in our existing 1.7.1 tag in GitHub.